### ISSUE TITLE
Use study-level parameters for participant compensation/communication to improve family and researcher communication

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-api#1314]
# TL;DR

Participant communication systems could use existing study-level parameters like study email/contact and study name and possibly participant-level parameters (child name) to improve the study communication process for both researchers and families. This would allow the researcher to (1) specify a different email per study and (2) auto-fill/provide default text that gives families the context of the communication automatically rather than depending on researchers' time or forethought.

# Narrative

1. Change email addresses provided to participants in participant compensation/other messages to use the specific **study** email instead of the lab email.
    -Currently, this is sourced from the lab email, meaning that researchers cannot use different emails for different studies. This is particularly an issue for multi-lab collaborations like Project GARDEN which need a different email for each study so each team can communicate with their participants directly.
    -This allows researchers to provide families with the option to directly reply to an email that is monitored by the specific staff running a particular study.

2. Change emails sent to families to auto-populate the child name, study name, and PI contact for the study itself.
    -It should be clear to families which study and child for which they are receiving compensation/other communication.
    -I would be happy to draft suggestions on language for this after scoping is complete and how the default text itself would work and where it could be placed within the email (e.g. does it populate for the researcher and then they can edit it, is it shown to the researcher but not editable, is it not shown to the researcher at all).

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
As always, I'm more than happy to help troubleshoot/brainstorm specific ways to implement these changes.

**Comment by becky-gilbert** [comment]
@ianchandlercampbell I've moved the study email part of this issue into it's own issue: https://github.com/lookit/lookit-api/issues/1351 But I'll leave this issue open since there are other things here that the study-level email doesn't address.
